frontend/packages/client/src/components/ProposalCreate/StepOne/index.js

```diff
@@ -127,38 +136,195 @@ const StepOne = ({
               <Input
                 placeholder="Minimum Balance"
                 classNames="rounded-sm border-light p-3 column is-full"
-                conatinerClassNames="mt-4 mb-4"
+                containerClassNames="mt-4 mb-4"
                 register={register}
                 error={errors['minBalance']}
                 name="minBalance"
               />
               <Input
                 placeholder="Maximum Weight"
                 classNames="rounded-sm border-light p-3 column is-full"
-                conatinerClassNames="mb-4"
+                containerClassNames="mb-4"
                 register={register}
                 error={errors['maxWeight']}
                 name="maxWeight"
               />
             </>
           )}
         </div>
+        <div className="border-light rounded-lg columns is-flex-direction-column is-mobile m-0 p-6 mb-6">
+          <h4 className="title is-5 mb-2">Type of Voting</h4>
+          <p className="has-text-grey mb-5">
+            Select the type of voting you would like to use for this proposal.
+            To learn more about these options,{' '}
+            <a
+              target="_blank"
+              rel="noreferrer noopener"
+              href="https://docs.cast.fyi"
+              className="is-underlined has-text-grey"
+            >
+              check out our FAQ
+            </a>
+            .
+          </p>
+          <div>
+            <div
+              className={`border-light rounded-sm is-flex is-align-items-center m-0 p-0 mb-4 cursor-pointer ${
+                voteType === 'single-choice' ? 'border-grey' : 'border-light'
+              }`}
+              onClick={() => handleVoteType('single-choice')}
+            >
+              <div className="p-4">
+                <div className="is-flex is-align-items-center mr-2">
+                  <div className="is-flex is-align-items-center mr-2">
+                    <label className="radio is-flex">
+                      <input
+                        {...register('voteType')}
+                        type="radio"
+                        value="single-choice"
+                        className="green-radio"
+                      />
+                      <span />
+                    </label>
+                  </div>
+                </div>
+              </div>
+              <div className="py-5 pr-5">
+                <h5 className="title is-6 mb-2">Single Choice Voting</h5>
+                <p>
+                  Voters can only vote on one option and all options are
+                  customized by proposal creator.
+                </p>
+              </div>
+              <div className="has-background-light-grey p-4 rounded-sm-br rounded-sm-tr is-flex is-flex-direction-column is-align-self-stretch is-justify-content-center ">
```

Comment:
```suggestion
              <div className="has-background-light-grey p-4 px-2-mobile rounded-sm-br rounded-sm-tr is-flex is-flex-direction-column is-align-self-stretch is-justify-content-center">
```
<img width="324" alt="Screen Shot 2022-10-14 at 10 43 44" src="https://user-images.githubusercontent.com/7526740/195861863-350060bd-3a77-463b-ab0b-fb533f48e661.png">